src/components/SQForm/useFormButton.tsx

```diff
@@ -3,7 +3,21 @@ import {useFormikContext} from 'formik';
 import {useDebouncedCallback} from 'use-debounce';
 import {hasUpdated} from '../../utils';
 
-export function useFormButton(isDisabled, shouldRequireFieldUpdates, onClick) {
+type UseFormButtonReturnType = {
+  isButtonDisabled: boolean;
+  hasFormBeenUpdated: boolean;
+  values: unknown;
+  initialValues: unknown;
+  isValid: boolean;
+  handleClick: (...args: unknown[]) => void;
```

Comment:
Also this type needs to be updated since handleClick has changed. If you hover over the `handleClick` declaration it should show you the correct inferred type. Probably something like this
```suggestion
  handleClick: DebouncedState<(event: React.MouseEventHandler<HTMLButtonElement>)=> void | undefined>;
```